CorsixTH/Lua/rooms/ward.lua

```diff
@@ -122,11 +122,29 @@ function WardRoom:doStaffUseCycle(humanoid)
   humanoid:queueAction(MeanderAction():setLoopCallback(meanders_loop))
 end
 
+-- This function is called once per tick per ward, unless there's no patient in the ward
+function WardRoom:getHealingAmount()
+  self.patient_idx = 0 -- Will be reset to nil after handling the last patient each tick
```

Comment:
If for some reason it stays 0 (no patients or all are leaving).

Reply:
Please if you introduce member variables, document their existence in the constructor, instead of randomly adding new state throughout all functions. That is, in the constructor say
``` lua
self.patient_idx = nil -- With descriptive meaning of the variable.
```
I don't like the variable name btw, "index in patients" doesn't say anything about the purpose.

What happens with the index if a patient leaves?